.github/dependabot.yaml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
```

Comment:
isn't it technically pipenv?

Reply:
Didn't see that option [here](https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem), was hoping that it will work with pipenv too